tests/unit/test_cli.py

```diff
@@ -28,6 +33,163 @@ def tearDown(self):
         m.db.session.remove()
         m.db.drop_all()
 
+    #################################
+    # ara host <cmd>
+    #################################
+    def test_host_list(self):
+        ctx = ansible_run()
+
+        cmd = ara.cli.host.HostList(None, None)
+        parser = cmd.get_parser('test')
+        args = parser.parse_args([])
+        res = cmd.take_action(args)
+
+        self.assertEqual(res[1][0][0], ctx['host'].id)
+
+    def test_host_list_for_playbook(self):
+        ctx = ansible_run()
+
+        cmd = ara.cli.host.HostList(None, None)
+        parser = cmd.get_parser('test')
+        args = parser.parse_args(['--playbook', ctx['playbook'].id])
+        res = cmd.take_action(args)
+
+        self.assertEqual(res[1][0][0], ctx['host'].id)
+
+    def test_host_list_for_non_existing_playbook(self):
+        ctx = ansible_run()
+
+        cmd = ara.cli.host.HostList(None, None)
+        parser = cmd.get_parser('test')
+        args = parser.parse_args(['--playbook', 'foo'])
+        res = cmd.take_action(args)
+
+        self.assertEqual(res[1], [])
+
+    def test_host_show_by_id(self):
+        ctx = ansible_run()
+
+        cmd = ara.cli.host.HostShow(None, None)
+        parser = cmd.get_parser('test')
+        args = parser.parse_args([ctx['host'].id])
+        res = cmd.take_action(args)
+
+        self.assertEqual(res[1][0], ctx['host'].id)
+
+    def test_host_show_by_name(self):
+        ctx = ansible_run()
+
+        cmd = ara.cli.host.HostShow(None, None)
+        parser = cmd.get_parser('test')
+        args = parser.parse_args([ctx['host'].name])
+        res = cmd.take_action(args)
+
+        self.assertEqual(res[1][0], ctx['host'].id)
+
+    def test_host_show_for_non_existing_host(self):
+        ctx = ansible_run()
+
+        cmd = ara.cli.host.HostShow(None, None)
+        parser = cmd.get_parser('test')
+        args = parser.parse_args(['foo'])
+
+        self.assertRaises(RuntimeError, cmd.take_action, args)
```

Comment:
I think this is easier to read as:

    with self.assertRaises(RuntimeError):
        cmd.take_action(args)